src/js/popup.js

```diff
@@ -346,6 +362,12 @@ function registerToggleHandlers() {
  * @param {Integer} tabId The id of the tab
  */
 function refreshPopup() {
+
+  //If there is any saved error text, fill the error input with it.
```

Comment:
Should probably move this paragraph down a bit, after the special/no tab data check if/else. Just for readability.

Would activate the overlay here (probably ...) too in order to reopen the popup straight into error reporting UI when the user left the popup with some error description text already entered.